app/scripts/controllers/overview.js

```diff
@@ -1153,6 +1153,18 @@ function OverviewController($scope,
         });
       });
     });
+
+    // sorting the bindingList for each instance by the first associated app for each binding.
+    // doing it once here eliminates the need to resort them throughout the console.
+    overview.bindingsByInstanceRef = _.reduce(overview.bindingsByInstanceRef, function(result, bindingList, key) {
+      result[key] = _.sortBy(bindingList, function(binding) {
+        var apps =  _.get(state.applicationsByBinding, [binding.metadata.name]);
+        var sorted = _.sortBy(apps, 'metadata.name');
```

Comment:
It would be better to sort the applicationsByBinding list above instead of here. Then it's also sorted when expanding the provisioned service row in the overview.